webviz_subsurface/_datainput/opm_init_io.py

```diff
@@ -278,6 +331,8 @@ def from_ecl_init_file(ecl_init_file: EclFile) -> Optional["Oil"]:
         if raw.num_tables == 0:
             return None
 
+        # If there is no Rs index, no gas is dissolved in the oil -> dead oil
+        # Otherwise, gas is dissolved and the ratio is given -> live oil
```

Comment:
Looking at the manual, there are also some other edge cases like PVCO (live oil in compressibility form) and PVCDO (dead oil in compressibility form). I don't think it is much in use, so this is probably ok for now. But I guess we should test these scenarios in the future.